### PR TITLE
Add URL encode/decode <action> for scenarios.

### PR DESCRIPTION
**Pull request by ticpu**
I really need this to decode the URI encoded Call-ID in the Replaces: header of a 3PCC scenario to send it via `<sendCmd>`.

### CODE REVIEW

File: docs/scenarios/actions.rst [hunk 1]
```diff
@@ -482,6 +482,24 @@ and microseconds since the epoch. For example::
...
++++++++++++++++++++++
+
+The urlencode and urldecode actions will replace the content of the
+variable specified in assign_to with the coded version. For example::
+
+Content of variable2 is "this: is a string".
+
+    <nop>
+      <action>
+        <urlencode assign_to="variable2" />
```
Comment (wdoekes) [inline docs/scenarios/actions.rst:495]: This PR looks awfully complete :+1: 

I just wonder about this `assign_to`. Doesn't it make more sense to call it `variable`?
```xml
<urlencode variable="variable2" />
```

**Review by wdoekes — CHANGES_REQUESTED** [review]

File: docs/scenarios/actions.rst (hunk 1 above)
Comment (ticpu) [reply docs/scenarios/actions.rst:495]: Absolutely, I based my work on the trim method, that explains why I used assign_to :)
I'll give it a quick change.

File: docs/scenarios/actions.rst [hunk 2]
```diff
@@ -482,6 +482,24 @@ and microseconds since the epoch. For example::
 
 
 
+urlencode / urldecode
++++++++++++++++++++++
+
+The urlencode and urldecode actions will replace the content of the
+variable specified in assign_to with the coded version. For example::
+
+Content of variable_to_be_encoded is "this: is a string".
```
Comment (wdoekes) [inline docs/scenarios/actions.rst:491]: Too fast ;)

File: docs/scenarios/actions.rst [hunk 3]
```diff
@@ -482,6 +482,24 @@ and microseconds since the epoch. For example::
 
 
 
+urlencode / urldecode
++++++++++++++++++++++
+
+The urlencode and urldecode actions will replace the content of the
+variable specified in assign_to with the coded version. For example::
```
Comment (wdoekes) [inline docs/scenarios/actions.rst:489]: Too fast ;)

Comment (ticpu) [reply docs/scenarios/actions.rst]: Oops, too fast indeed, I didn't see your comment before requesting re-review. Fixed.

**Pull request merged**